drivers/sd/bcm2836/bcm2836sdhc/bcm2836sdhc.c

```diff
@@ -455,19 +455,15 @@ SdhcSlotInitialize (
     SdhcExtension->OutstandingRequest = NULL;
 
     //
-    // Enable all interrupt signals from controller to the OS,
-    // but mask all.
-    // Means we are only using SDHC_INTERRUPT_ERROR_STATUS_ENABLE to
-    // control interrupts, this way disabled events do not get reflected
-    // in the status register.
+    // Disable interrupts until we're ready to handle them.
```

Comment:
> Note that this can be considered a bug in sdport - it should be ready
> for interrupts before it connects the interrupt. I've provided this
> feedback to the sdport owners. Fixing the driver works around the issue.

Then marking this as a "workaround" in the comment would be a good idea.